atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/atomic/ConsensusForgettingStore.java

```diff
@@ -33,15 +33,16 @@
  *   value until a put occurs.
  */
 public interface ConsensusForgettingStore {
+
     /**
      * An atomic put unless exists operation. If this method throws an exception, there are no consistency guarantees:
```

Comment:
This makes sense - the semantics are still the same (and so these docs) are OK, if I'm reading this correctly. That said, do we plan to change the semantics of this method in the future, maybe for some implementations?

Reply:
More generally, will the logic for making the state change from "started" to the start of the transactions3 protocol happen at this layer, or somewhere else?